documents/description.md

```diff
@@ -1,9 +1,41 @@
+
+Other sources of error in the project could be in querying the database and searching for particular wines. If there are no wines available that match a certain set of criteria, the program should produce appropriate error messages and not shut down. It is also important in designing the language to determine which aspects of a wine description should be optional. Some users may have nothing to write for secondary aromas on a particular wine. In the spirit of the project, this should be completely reasonable. However, if this is an optional attribute, it should be handled by the system as such and the language should not throw errors if the field is left blank.
+
 
 ## Example computations
+*Describe some example computations in your DSL. These computations should describe what happens when a specific program in your DSL executes. Note that you shouldn't describe the syntax of the program. Rather you should describe some canonical examples that help someone else understand the kinds of things that your DSL will eventually be able to do.*
+
+Let's run through an example of how a user would use the project from end-to-end. Say that our sample user knows a bit about wines but wants to improve their knowledge. Tonight they have chosen to drink a Sauvignon Blanc. Once they have a glass, they begin writing their tasting notes on the wine. They report the grape varietal, the aromas/tastes of the wine, an individual score for the wine, and their own personal opinions on the wine. At any point while writing their tasting notes, they can search for this wine and include the exact wine as part of the notes. After they finish writing, they can save the notes to their own personal wine cellar and continue enjoying their glass of wine. 
+
+If the user wants to learn how their opinions compare to those of wine professionals, they can search for the wine in their cellar and run a simple command to find professional tasting notes on the wine. They can then see a side-by-side comparison of their notes versus what a professional would have said if they had written notes in the same format. The user can repeat this process for any wine in their cellar. Satisfied that their opinions loosely match with those of the professionals, the user would continue to enjoy their night and their wine.
+
+Depending on the timeline of the project, I might add functionality for users to compare two wines in their cellar with one another (as opposed to professional notes). This could be beneficial if a user wants to see if their impressions of similar wines would be comparable.
```

Comment:
Don't want to push too much on this issue, but again, think of the problem each part of your DSL is solving and make sure it it both a problem the user in the domain actually has and is big enough to warrant part of the DSL and that the DSL is solving the problem correctly. 

Another question I have for this part is, how does a user know if two wines are similar?